runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1271,6 +1305,53 @@ gcParseReconfigurableArguments(J9JavaVM* vm, J9VMInitArgs* args)
 
 	return true;
 
+_error:
+	return false;
+}
+/**
+ * Parse command line arguments used ONLY for CRIU restore. Note the arguments
+ * are now passed in through an option file.
+ *
+ * @return true if parsing was successful, false otherwise.
+ */
+bool
+gcParseReconfigurableCommandLine(J9JavaVM* vm, J9VMInitArgs* args)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(vm);
+	IDATA index = -1;
+	IDATA result = 0;
+	const char *memoryOption = NULL;
```

Comment:
we can remove this and  just use the macro instead (VMOPT_XSOFTMX)
furthermore, for some reason we have a duplicate macro OPT_SOFTMX, defined and used in this very GC file, so it probably makes sense to use that one again